### DISCUSSION TITLE
Allowed Hostnames does not apply to all screenshot widths

### DISCUSSION THREAD

**Discussion by j-waaang** [percy/cli#555] [Support]
I'm using percy/storybook and this is my percy config:
```
module.exports = {
  version: 2,
  snapshot: {
    widths: [375, 1280],
  },
  discovery: {
    'allowed-hostnames': ['domain.com', 'www.domain.com', 'www2.domain.com'],
  },
}
```
My stories have images on domains in 'allowed-hostnames'. On the screenshots taken by percy I can only see the images on width 375 screenshots. On 1280 I only see broken image icons and the image title.

If I only have 1280 inside widths, then I can see my images on the screenshots.

Could it be that the parameters of 'allowed-hostnames' are only applied to the first element inside widths?

## Environment
- "@percy/cli": "^1.0.0-beta.65",
- "@percy/storybook": "4.0.1",
- node14

**Comment by wwilsman** [comment]
Hi @namelesswc!

This wouldn't be an allowed-hostnames issue causing this, as that is set globally for the page and not per-width.

However, this does sound suspiciously like issue #511. The fix has been merged and I just cut a new release. Try out `beta.66` to see if it helps here 🙂

**Comment by j-waaang** [comment]
Just tried it with `beta.67` and it did not work. I will try to create a minimal reproduceable project on codepen when I come back from vacation.
